docs/modules/ROOT/pages/tmail-backend/features/openpaas-integration.adoc

```diff
@@ -0,0 +1,34 @@
+= OpenPaaS Integration
+:navtitle: OpenPaaS Integration
```

Comment:
Is this new page integraed into https://github.com/linagora/tmail-backend/blob/master/docs/modules/ROOT/nav.adoc ?